deploy/auto-install.sh

```
@@ -190,19 +187,19 @@ upgrade_docker_openwisp() {
 
 	download_docker_openwisp "$openwisp_version"
 
-	cd $INSTALL_PATH &>>$LOG_FILE
+	cd "$INSTALL_PATH" &>>"$LOG_FILE"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)